tetrateci/manifest.yaml.in

```diff
@@ -0,0 +1,49 @@
+version: $TAG
+
+docker: $HUB
+
+directory: /tmp/istio-release
+
+ignoreVulnerability: true
+
+dependencies:
+  istio:
```

Comment:
This has to be the local path of our istio fork

Reply:
Ahh right